core/lib/spree/app_configuration.rb

```diff
@@ -217,6 +217,10 @@ class AppConfiguration < Preferences::Configuration
     #   @return [Integer] Promotions to show per-page in the admin (default: +15+)
     preference :promotions_per_page, :integer, default: 15
 
+    # @!attribute [rw] disable_actionless_promotion_validation
+    #   @return [Boolean] Promotions should have actions associated before activation (default: +true+)
```

Comment:
I'd  use `being considered active` to be consistent with the language used in the rest of the PR. `activation` makes me think that there's something to activate with an action.